src/pages/settings/group-settings/common/hooks/useConfigureGroupSettings.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { activeGroupSettingsAtom } from '$app/common/atoms/settings';
+import { useSetAtom } from 'jotai';
+import { useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { updateChanges } from '$app/common/stores/slices/company-users';
+import { setActiveSettings } from '$app/common/stores/slices/settings';
+import { GroupSettings } from '$app/common/interfaces/group-settings';
+
+interface Params {
+  withoutNavigation: boolean;
+}
+export const useConfigureGroupSettings = (params?: Params) => {
```

Comment:
`function` style of declaration, please.

Reply:
Changed!